zerver/views/auth.py

```diff
@@ -330,6 +330,8 @@ def login_page(request, **kwargs):
     if is_subdomain_root_or_alias(request) and settings.REALMS_HAVE_SUBDOMAINS:
         redirect_url = reverse('zerver.views.registration.find_my_team')
         return HttpResponseRedirect(redirect_url)
+    if request.user.is_authenticated():
```

Comment:
I think it doesn't matter functionally since you can't login to the root subdomain, but I think it reads better with this first, so let's move it there.